resources/osl_mysql_test.rb

```diff
@@ -10,16 +10,22 @@
 property :server_password, String, default: 'osl_mysql_test'
 property :encoding, String, default: 'utf8mb4'
 property :collation, String, default: 'utf8mb4_unicode_ci'
+property :version, [String, nil]
 property :database_parameters, Hash, default: {}
 
 # Install the mariadb package, set up the service, set up the user, then set up the given database
 action :create do
-  # Install the package, and set up the service
+  # Setup the epel repo if we are installing from MariaDB
+  include_recipe 'osl-repos::epel' if new_resource.version
```

Comment:
Why does this need to be implicitly added here? Please add a little more to the comment.